src/main.rs

```diff
@@ -42,44 +39,6 @@ struct Args {
     stellar_node_database_url: Option<String>,
 }
 
-fn setup_logger() {
-    let stdout = ConsoleAppender::builder().build();
-
-    let config = Log4rsConfig::builder()
-        .appender(Appender::builder().build("stdout", Box::new(stdout)))
-        .build(Root::builder().appender("stdout").build(LevelFilter::Info))
-        .unwrap();
-
-    log4rs::init_config(config).unwrap();
-}
-
-fn setup_configuration(args: Args) -> Configuration {
-    let mut config_builder = Config::builder()
```

Comment:
We might want to use [Figment](https://docs.rs/figment/latest/figment/) here, which can also handle args without the overrides. Not critical to implement though.

Reply:
Wasn't familiar with it. I'll check it out.